executor.go

```diff
@@ -82,45 +82,38 @@ type ExecutionContext struct {
 
 func buildExecutionContext(p BuildExecutionCtxParams) (*ExecutionContext, error) {
 	eCtx := &ExecutionContext{}
-	operations := map[string]ast.Definition{}
+	//	operations := map[string]ast.Definition{}
```

Comment:
@chris-ramon Thanks for catching that! I've removed a couple of other commented out unused vars as well 👍